src/styles/main.scss

```diff
@@ -1,3 +1,46 @@
-body {
-  margin: 0;
+@import "./_variables/reset";
+@import "./_variables/variables";
+
+.calendar {
```

Comment:
Please add this 
![image](https://user-images.githubusercontent.com/54936044/229475887-0acb11f6-511f-410e-a50a-97d7d47eb249.png)

Reply:
done